mantle/harness/harness.go

```diff
@@ -34,6 +34,9 @@ import (
 	"github.com/coreos/mantle/harness/testresult"
 )
 
+const DefaultTimeoutFlag = 0
+const defaultTimeout = 10
```

Comment:
```suggestion
const defaultTimeout = 10 * time.Minute
```

So it's more self-describing *and* it's already a time value.